zebrad/src/components/mempool.rs

```diff
@@ -51,7 +53,7 @@ pub struct Mempool {
     ///
     /// ##: Correctness: only components internal to the [`Mempool`] struct are allowed to
     /// inject transactions into `storage`, as transactions must be verified beforehand.
-    storage: storage::Storage,
+    pub storage: storage::Storage,
```

Comment:
It is a shame to make this public just for the tests, i might think on something better tomorrow for this. Ideas are welcome.

Reply:
One alternative is to add a getter method that's only compiled for `cfg(test)`:

```
impl Mempool {
    #[cfg(test)]
    pub fn storage(&mut self) -> &mut storage::Storage {
        &mut self.storage
    }
}
```

Reply:
Yea, definitly better. i Did this at https://github.com/ZcashFoundation/zebra/pull/2720/commits/40f98758fe0e9621ff861876bcedaca3be04106c